marshall/package.json

```diff
@@ -6,6 +6,7 @@
   "dependancies": {
     "gulp": "3.9.1",
     "chai": "3.5.0",
-    "mocha": "2.4.0"
+    "mocha": "2.4.0",
+    "eslint": "2.9.0"
   }
 }
```

Comment:
You should make your package.json with npm init instead of just making it yourself.  You are missing some things and you have misspelled some of your fields in here.